cli/cluster/cmd/publish.go

```diff
@@ -93,9 +100,9 @@ func publishCtxValidateConfig(publishCtx PublishCtx, instance string) error {
 	return nil
 }
 
-// replaceInstanceConfig replaces an instance configuration in the collected
+// replaceOrCreateInstanceConfig replaces or creates an instance configuration in the collected
 // cluster configuration and republishes it.
-func replaceInstanceConfig(instance string, config *libcluster.Config,
+func replaceOrCreateInstanceConfig(group, replicaset, instance string, config *libcluster.Config,
```

Comment:
It's better to split the function into 2 or 3 then to name it as it. Or choose an another name for the function.

Reply:
Renamed to `setInstanceConfig`.